lib/libutils/ext/include/compiler.h

```diff
@@ -53,4 +53,8 @@
 #define __rodata	__section(".rodata")
 #define __rodata_unpaged __section(".rodata.__unpaged")
 
+#define __compiler_bswap64(x)	__builtin_bswap64((x))
```

Comment:
Yes, that's why I use these `__compiler_bswapxx` instead to make it easier to port to another compiler. More or less all the stuff in this file is gcc specific.